$OpenBSD$

disable block using x11_sym that seems to be a part of the bundled SDL

Index: src/gui/sdl_mapper.cpp
--- src/gui/sdl_mapper.cpp.orig
+++ src/gui/sdl_mapper.cpp
@@ -3668,7 +3668,7 @@ void BIND_MappingEvents(void) {
 # endif
 #endif
 #if defined(SDL_VIDEO_DRIVER_X11)
-# if defined(C_SDL2)
+# if defined(C_SDL2) || defined(__OpenBSD__)
 # else
                 {
                     char *LinuxX11_KeySymName(Uint32 x);
